api-admin/src/main/java/bio/terra/pearl/api/admin/service/kit/KitExtService.java

```diff
@@ -72,9 +71,6 @@ public Collection<KitRequest> getKitRequests(AdminUser adminUser, String enrolle
 
   public void refreshKitStatuses(
       AdminUser adminUser, String portalShortcode, String studyShortcode) {
-    if (!adminUser.isSuperuser()) {
```

Comment:
Batch status fetching is intentionally done during non-busy hours for DSM to avoid the chance of overwhelming it. I think we can rely on us as superusers to understand that and weigh the risk of consequences, but I don't expect study staff to know, nor should they care. If you want to open this function up to study staff, please discuss with Pegah and the DSM team.

Instead, the idea was to expose (probably in the individual enrollee kit view) the date/time that the current known status was fetched and allow study staff to refresh the status for that one kit (for which there is already a service function, but not yet an API endpoint).

Reply:
As a compromise, can we add the UI enhancement while keeping it superuser-only for now until we can have a conversation about it with the DSM team?

Reply:
good idea